src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1263,6 +1301,8 @@ static SvgNode* _createNode(SvgNode* parent, SvgNodeType type)
     node->style->stroke.join = StrokeJoin::Miter;
     node->style->stroke.scale = 1.0;
 
+    node->style->paintOrder = _toPaintOrder("fill stroke");
```

Comment:
I think it is better to write true than `_toPaintOrder("fill stroke")` because "fill storke" always returns `true`.

Reply:
my motivation was to make the setting the def values to be as independent as possible from the paintOrder variable. to be honest it was for me very hard to think about paint-order without markers :D I kept them in my mind all the time, that's why maybe this was more complicated than it should be